addons/account/wizard/account_move_send_views.xml

```diff
@@ -15,16 +15,20 @@
                 <field name="enable_download" invisible="1"/>
                 <field name="enable_send_mail" invisible="1"/>
                 <field name="send_mail_readonly" invisible="1"/>
+                <field name="send_mail_warning_message" invisible="1"/>
                 <field name="display_mail_composer" invisible="1"/>
                 <field name="mail_lang" invisible="1"/>
-                <field name="mail_partner_ids" widget="many2many_tags_email" class="d-none" context="{'force_email': True}"/>
 
                 <!-- Warnings -->
                 <div name="warnings">
-                    <div class="alert alert-warning"
-                        role="alert"
-                        invisible="not send_mail_warning_message">
-                        <field name="send_mail_warning_message"/>
+                    <span invisible="send_mail_warning_message != 'missing_info'"><b>E-mail</b></span>
+                    <div class="alert alert-warning" role="alert"
+                        invisible="send_mail_warning_message != 'missing_info'">
+                        Cannot send email because some
```

Comment:
We should probably refine the warning message a bit :thinking: 
In case of 'multi' we still send the email to valid partners.